src/components/MainPage/MainPage.js

```diff
@@ -8,13 +8,14 @@ class MainPage extends React.Component {
   }
 
   render() {
-    return (<div className={'welcome-screen ' + (this.state.enterAnimation ? "enter-game-animation " : "open-animation ")}>
-      <div className="title enter-game-animation-top">Hangman 2077</div>
-      <img src={require('./../../assets/hangman_home_page.gif')} className="hangman-gif" alt="test"></img>
-      <div >
-        <button className="button"
+    return (
+    <div className={'welcome-screen ' + (this.state.enterAnimation ? "enter-game-animation " : "open-animation ")}>
+      <div className="title enter-game-animation-top row justify-content-center">Hangman 2077</div>
+      <img src={require('./../../assets/hangman_home_page.gif')} className="hangman-gif row justify-content-center" alt="test"></img>
+      <div className="row justify-content-center">
```

Comment:
I would like to add a question, how does 'row' class work when it is not mentioned in MainPage.css? Also How 3 classNames are merged?